### DISCUSSION TITLE
Spatial averages

### DISCUSSION THREAD

**Discussion by makrandak** [Nek5000/nekRS#563] [Q&A] [answered]
Hello 

Is there an example where the entire spatial average (in all directions is computed)? I want to subtract the average of a field at every timestep from itself after every timestep.

Thank you for your time and help.

With sincere regards,
Makrand

**Comment by stgeke** [comment, marked as answer]
See e.g.:
https://github.com/Nek5000/nekRS/blob/next/examples/hit/hit.udf#L45

**Reply by makrandak** [reply]
Thank you ver much, @stgeke!
